### ISSUE TITLE
Error: connect EHOSTUNREACH

### ISSUE THREAD

**Issue by pedroserano** [kyleaa/homebridge-samsungtv2016#6]
I'm getting a weird error that crashes my homebridge running on a pi 3, is anyone else getting this? The IP below is my Samsung TV. After homebridge crashes, and I restart it, it seems to work as long as I have the TV on. I'm not sure if it's crashing just after a period of time, or after the TV is turned off.

> Accessory [Homebridge] Setting Characteristic "On" to value false +3ms
> events.js:141
>       throw er; // Unhandled 'error' event
>       ^
> 
> Error: connect EHOSTUNREACH 10.0.1.12:8001
>     at Object.exports._errnoException (util.js:870:11)
>     at exports._exceptionWithHostPort (util.js:893:20)
>     at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1062:14)

**Comment by pedroserano** [comment]
Unfortunately this is not unique to the samsungtv2016 plug-in: see, e.g. https://github.com/KraigM/homebridge-harmonyhub/issues/4

**Comment by kyleaa** [comment]
Can you try the latest build and see if you're still running into this? I can't imagine anything has changed that would prevent it, but there was some improvement to the logging that might provide some more value.

**Comment by hogendoorn** [comment]
Hi @kyleaa 

Just recieved this error last night. I am running the latest version

```
events.js:141
      throw er; // Unhandled 'error' event
      ^

Error: connect EHOSTUNREACH 192.168.16.80:8001
    at Object.exports._errnoException (util.js:837:11)
    at exports._exceptionWithHostPort (util.js:860:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1060:14)
```
Can i retrieve any more logs ?

**Comment by kyleaa** [comment]
I've published version 0.0.7 that should handle that websocket error event. Thanks @inzlabs for providing more detailed logs. This error was occurring when a Power Off command was sent for a TV that was already off - I'm not sure why the homekit client wasn't checking the TV status first.

**Comment by bachoo786** [comment]
Hi 

I have managed to install homebridge and was wondering how does this software work? What are the next steps involved? Also can this be integrated with homeassistant?
